lib/suspenders/adapters/heroku.rb

```diff
@@ -0,0 +1,52 @@
+module Suspenders
+  module Adapters
+    class Heroku
+      def initialize(app_builder)
+        @app_builder = app_builder
```

Comment:
You mean a private reader? I usually do that, but as it is not a well establish convention (there is actually a warning against it in ruby), I didn't do that. Shall I move to that?